### PR TITLE
create the com.ibm.ws.jaxws.2.3.web project for the jaxws-2.3 feature

### PR DESCRIPTION
**Pull request by WhiteCat22**
create the com.ibm.ws.jaxws.2.3.web project for the jaxws-2.3 feature
#7650

### CODE REVIEW

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
green build

File: dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+bVersion=1.0
+
+Bundle-Name: JAX-WS 2.3 Web
+Bundle-SymbolicName: com.ibm.ws.jaxws.2.3.web
+Bundle-Description: IBM JAX-WS 2.3 Web support; version=${bVersion}
+
+-dsannotations: com.ibm.ws.jaxws.web.WebXMLEndpointInfoConfigurator
+
+Service-Component: \
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd]: To-Do: Convert the Service-Components to DS Annotation components

File: dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+     properties:="service.vendor=IBM"
+
+Export-Package: \
+    com.ibm.ws.jaxws23.web
+
+Import-Package: \
+   com.ibm.wsspi.adaptable.module.adapters, \
+   com.ibm.wsspi.webcontainer.collaborator, \
+   com.ibm.ws.container.service.metadata, \
+   javax.xml.ws.*;version="[2.2,3)", \
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd]: Update javax.xml.ws.* and org.apache.cxf.* to 2.3 and 3.3 respectively

File: dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+   *
+
+Private-Package:\
+   com.ibm.ws.jaxws23.web.*
+
+instrument.classesExcludes: com/ibm/ws/jaxws23/web/internal/resources/*.class
+
+-buildpath: \
+	javax.jws:jsr181-api;version=latest,\
+	com.ibm.websphere.javaee.jaxws.2.2;version=latest,\
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd]: To-Do: Pull in new com.ibm.websphere.javee.jaxws.2.3 bundle

File: dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    org.apache.cxf.cxf-api;strategy=exact;version=2.6.2.ibm-s20190122-2227,\
+	org.apache.cxf.cxf-rt-bindings-soap;strategy=exact;version=2.6.2.ibm-s20180529-1900,\
+	org.apache.cxf.cxf-rt-core;strategy=exact;version=2.6.2.ibm-s20180529-1900,\
+	org.apache.cxf.cxf-rt-databinding-jaxb;strategy=exact;version=2.6.2.ibm-s20170216-1739,\
+	org.apache.cxf.cxf-rt-frontend-jaxws;strategy=exact;version=2.6.2.ibm-s20180914-1510,\
+	org.apache.cxf.cxf-rt-frontend-simple;strategy=exact;version=2.6.2.ibm-s20170216-1739,\
+	org.apache.cxf.cxf-rt-management;strategy=exact;version=2.6.2.ibm-s20170216-1739,\
+	org.apache.cxf.cxf-rt-transports-http;strategy=exact;version=2.6.2.ibm-s20180821-1653,\
+	org.apache.cxf.cxf-rt-ws-policy;strategy=exact;version=2.6.2.ibm-s20190515-1254,\
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd]: To-Do: Update all the CXF jars to the new 3.3 versions

File: dev/com.ibm.ws.jaxws.2.3.web/.settings/org.eclipse.jdt.core.prefs [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+org.eclipse.jdt.core.codeComplete.fieldPrefixes=
+org.eclipse.jdt.core.codeComplete.fieldSuffixes=
+org.eclipse.jdt.core.codeComplete.localPrefixes=
+org.eclipse.jdt.core.codeComplete.localSuffixes=
+org.eclipse.jdt.core.codeComplete.staticFieldPrefixes=
+org.eclipse.jdt.core.codeComplete.staticFieldSuffixes=
+org.eclipse.jdt.core.codeComplete.staticFinalFieldPrefixes=
+org.eclipse.jdt.core.codeComplete.staticFinalFieldSuffixes=
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.web/.settings/org.eclipse.jdt.core.prefs]: Change to 1.8

File: dev/com.ibm.ws.jaxws.2.3.web/.classpath [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.3.web/.classpath]: Change to 1.8

File: dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd (hunk 2 above)
Comment (WhiteCat22) [reply dev/com.ibm.ws.jaxws.2.3.web/bnd.bnd]: I decided to update these in a future PR after the project has been created and we start using it

**Comment by WhiteCat22** [comment]
#build to make sure addressing Thomas's changes don't break anything

**Comment by WhiteCat22** [comment]
#build to address changes

**Comment by skasund** [comment]
L2 message review complete

File: dev/com.ibm.ws.jaxws.2.3.web/resources/com/ibm/ws/jaxws/web/internal/resources/JaxWsWebMessages.nlsprops [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+#COMPONENTNAMEFOR CWWKW WebSphere JAXWS 2.3 Web Code
+#RANGE 0400 - 0499
+#
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+#{0} is the servlet name
+error.urlPattern=CWWKW0400E: The url-pattern of the servlet-mapping for servlet {0} must not contain an asterisk (*).
+error.urlPattern.explanation=The url-pattern of the servlet-mapping must be an exact match pattern and must not contain an asterisk.
+error.urlPattern.useraction=Make sure that url-pattern of servlet-mapping of the servlet is an exact match pattern.
+
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.jaxws.2.3.web/resources/com/ibm/ws/jaxws/web/internal/resources/JaxWsWebMessages.nlsprops]: Looks like it no changes are needed.

Comment (WhiteCat22) [reply dev/com.ibm.ws.jaxws.2.3.web/resources/com/ibm/ws/jaxws/web/internal/resources/JaxWsWebMessages.nlsprops]: thanks!

Comment (WhiteCat22) [reply dev/com.ibm.ws.jaxws.2.3.web/resources/com/ibm/ws/jaxws/web/internal/resources/JaxWsWebMessages.nlsprops]: FYI, in a future PR we are considering reusing the JAXWS messages, so these MAY get pulled out.

**Pull request merged**